### DISCUSSION TITLE
Search bar should be centered

### DISCUSSION THREAD

**Discussion by Averasia** [valinet/ExplorerPatcher#3086] [Ideas]
The more icons there are on the taskbar, the search bar starts to move to the right, it should always stay in the middle.

**Comment by pyrates999** [comment]
Which taskbar is this? windows 10 or 11?

**Reply by Averasia** [reply]
Windows 10

**Reply by pyrates999** [reply]
And are you running windows 10 or 11?

**Reply by Averasia** [reply]
I used to use 10 so I wrote it here, now I use 11

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Reply by Averasia** [reply]
I'm using Windows 11 now, so I don't use Explorer patcher, I don't need it anymore. I was using it to put the taskbar in the middle, it's already in the middle in Windows 11.

**Reply by pyrates999** [reply]
Please close this then.
